static/js/settings_emoji.js

```diff
@@ -52,13 +52,37 @@ exports.reset = function () {
     meta.loaded = false;
 };
 
+function emoji_name_sort(a, b) {
+    if (a.name > b.name) {
+        return 1;
+    } else if (a.name === b.name) {
+        return 0;
+    }
+    return -1;
+}
+
+
+exports.sort_but_user_emoji_on_top = function () {
```

Comment:
This function ignores its argument, which seems like a bug.  

Why not just define a comparison function and just sort using that?